infrastructure/data/index.ts

```diff
@@ -39,6 +39,7 @@ const db = new aws.rds.Instance("app", {
   username: DB_ROOT_USERNAME,
   password: config.require("db-password"),
   skipFinalSnapshot: true,
+  snapshotIdentifier: "after-postgres-update-167",
```

Comment:
I imagine that once a snapshot is "restored", its' name is no longer a valid `snapshotIdentifier` - so I guess if this succeeds, we'll want to add a "clean up" step to revert this line & commit file back to original state so it can be successfully deploying againt _next time_? That's the only part of doing a restore via Pulumi that isn't quite clicking for me yet!

Reply:
@jessicamcinchak a great quesion, I also wonder this! will find out shortly 😁

Reply:
(my guess is that Pulumi knows the instance is derived from a particular snapshot, so doesn't try and rebuild it every time??)